### PR TITLE
[Frontend][TFLite] Add MIRROR_PAD operator

### PR DESCRIPTION
**Pull request by wyc-ruiker**
Thanks for contributing to TVM!   Please refer to guideline https://docs.tvm.ai/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/apache/incubator-tvm/blob/master/CONTRIBUTORS.md#reviewers) by @ them in the pull request thread.

@FrozenGene @inadob @anijain2305 @apivovarov Could you please help to review this patch.

### CODE REVIEW

File: python/tvm/relay/frontend/tflite.py [hunk 1]
```diff
@@ -1422,10 +1423,48 @@ def convert_pad(self, op):
...
+        try:
+            from tflite.Operator import Operator
+            from tflite.BuiltinOptions import BuiltinOptions
+            from tflite.MirrorPadOptions import MirrorPadOptions
+        except ImportError:
+            raise ImportError("The tflite package must be installed")
+
+        # the quantized form MirrorPad is not yet implemented in TFLite.
+        if self.is_quantized(op):
+            raise tvm.error.OpNotImplemented(
```
Comment (FrozenGene) [inline python/tvm/relay/frontend/tflite.py]: Consider supporting for quantization, it shouldn’t be hard and be careful the pad value.

Comment (wyc-ruiker) [reply python/tvm/relay/frontend/tflite.py]: I have tried to support it. But when I write a test case like this:
```python
in_data = [array_ops.placeholder(shape=data[0].shape, dtype='float32', name='in')]
inq_data = [tf.quantization.fake_quant_with_min_max_args(in_data[0], min=-100, max=100, name="inq")]
out = array_ops.pad(inq_data[0], ops.convert_to_tensor(data[1], dtype=data[1].dtype), mode=mode)
out = tf.quantization.fake_quant_with_min_max_args(out, min=-100, max=100, name="out")
compare_tflite_with_tvm([data[0]], ['inq:0'], inq_data, [out], quantized=True)
```
I find this case check failed in TFlite:
```
2020-02-05 23:44:37.789418: F ./tensorflow/lite/toco/toco_tooling.h:38] Check failed: s.ok() Unimplemented: this graph contains an operator of type MirrorPad for which the quantized form is not yet implemented. Sorry, and patches welcome (that's a relatively fun patch to write, mostly providing the actual quantized arithmetic code for this op).
Fatal Python error: Aborted
Current thread 0x00007fee879dd700 (most recent call first):
  File "/home/wangyc/.local/lib/python3.7/site-packages/tensorflow/lite/toco/python/toco_from_protos.py", line 33 in execute
  File "/home/wangyc/.local/lib/python3.7/site-packages/absl/app.py", line 250 in _run_main
  File "/home/wangyc/.local/lib/python3.7/site-packages/absl/app.py", line 299 in run
  File "/home/wangyc/.local/lib/python3.7/site-packages/tensorflow/python/platform/app.py", line 40 in run
  File "/home/wangyc/.local/lib/python3.7/site-packages/tensorflow/lite/toco/python/toco_from_protos.py", line 59 in main
  File "/home/wangyc/.local/bin/toco_from_protos", line 8 in <module>
Aborted (core dumped)
```

Comment (inadob) [reply python/tvm/relay/frontend/tflite.py]: If TFLite doesn't support uint8 input then I don't think we can write qnn implemenation at that point.

Comment (FrozenGene) [reply python/tvm/relay/frontend/tflite.py]: Ok. Thanks for clarification. We could skip this implementation now as TFlite is not supported.

Comment (u99127) [reply python/tvm/relay/frontend/tflite.py]: For future reference, https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/mlir/lite/ir/tfl_ops.td#L2803

File: python/tvm/relay/frontend/tflite.py [hunk 2]
```diff
@@ -1422,10 +1423,48 @@ def convert_pad(self, op):
...
+                'TFlite quantized MIRROR_PAD operator is not supported yet.')
```
Comment (u99127) [inline python/tvm/relay/frontend/tflite.py]: Pedantry : A not implemented error suggests to me as a user that the support exists in the tflite tooling but the TVM stack has not yet implemented the support i.e. the tflite tooling can't yet produce the mirror_pad operation, do we need a different notification here actually asking the user to report an issue asking for a feature to be added ?

Comment (wyc-ruiker) [reply python/tvm/relay/frontend/tflite.py]: I think this exception will not be triggered when quantized MIRROR_PAD is not implemented by TFLite. If the exception is triggered, it certainly means the TVM stack doesn't support quantized MIRROR_PAD.

**Comment by FrozenGene** [comment]
@u99127 @inadob Please approve explicitly if you think this PR is good now.

**Comment by inadob** [comment]
LGTM

**Comment by FrozenGene** [comment]
> LGTM

@inadob Could you do the approve operation as this guide? https://docs.tvm.ai/contribute/code_review.html#approve-and-request-changes-explicitly Thanks!

**Comment by FrozenGene** [comment]
Thanks @wyc-ruiker @inadob It is merged now.

**Pull request merged**